src/components/Tags/index.js

```diff
@@ -100,12 +107,13 @@ const Tags = ({
       <input
         type='text'
         ref={inputRef}
-        className='mc-form-element__element'
+        className={inputClasses}
         onChange={handleInputChange}
         onFocus={handleInputFocus}
         onBlur={handleInputBlur}
         onKeyDown={handleKeyDown}
         size={value.length || 1}
```

Comment:
i wonder if adding the placeholder length here instead of additional CSS would be better?

```suggestion
         size={value.length || placeholder.length || 1}
```

Reply:
It works! Done 😃